data/sql/updates/pending_db_world/rev_1473111010225487800.sql

```diff
@@ -0,0 +1,130 @@
+INSERT INTO version_db_world(`sql_rev`) VALUES ('1473111010225487800');
+
+
+// Ambassitor Kelmer Walk fix

```

Comment:
I don't know if in some different database server applications it can affect, but is better to keep the same format